langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiChatModel.java

```diff
@@ -244,13 +252,22 @@ private Response<AiMessage> generate(List<ChatMessage> messages,
             options.setFunctionCall(new FunctionCallConfig(toolThatMustBeExecuted.name()));
         }
 
-        ChatCompletions chatCompletions = client.getChatCompletions(deploymentName, options);
-
-        return Response.from(
-                aiMessageFrom(chatCompletions.getChoices().get(0).getMessage()),
-                tokenUsageFrom(chatCompletions.getUsage()),
-                finishReasonFrom(chatCompletions.getChoices().get(0).getFinishReason())
-        );
+        try {
+            ChatCompletions chatCompletions = client.getChatCompletions(deploymentName, options);
+            return Response.from(
+                    aiMessageFrom(chatCompletions.getChoices().get(0).getMessage()),
+                    tokenUsageFrom(chatCompletions.getUsage()),
+                    finishReasonFrom(chatCompletions.getChoices().get(0).getFinishReason())
+            );
+        } catch (HttpResponseException httpResponseException) {
+            logger.info("Error generating response, {}", httpResponseException.getValue());
```

Comment:
minor: is logging required in this case? User will know about content filter from the returned `Response`
(this applies to all classes)